xgbgen/main.go

```diff
@@ -34,13 +36,8 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
-	if flag.NArg() != 1 {
-		log.Printf("A single XML protocol file can be processed at once.")
-		flag.Usage()
-	}
```

Comment:
I would prefer to see this more helpful context and information instead of a fatal log entry with the unhelpful message: `open: file does not exist: `

Reply:
Fixed by cbed66aa3ac97f6bb2f59ad18df2472a2d4368bd